argus/recipes/base.py

```diff
@@ -50,19 +50,19 @@ def __init__(self, instance_id, api_manager, remote_client, image,
         self._service_type = service_type
         self._output_directory = output_directory
 
-    def _execute(self, cmd):
-        """Execute and return only the standard output."""
+    def _execute(self, *args, **kwargs):
```

Comment:
It's better to write all the supported arguments upfront, since then it's self documenting at least.